CheckDrive.Web/CheckDrive.Web/Views/Cars/Index.cshtml

```diff
@@ -1,11 +1,16 @@
-﻿@model CheckDrive.ApiContracts.Car.CarDto
+﻿@using CheckDrive.ApiContracts.Car
+@model CheckDrive.ApiContracts.Car.CarDto
 
 @{
     ViewData["Title"] = "Index";
-    Layout = "~/Views/Shared/_Layout.cshtml";
+    // Layout = "~/Views/Shared/_Layout.cshtml";
 }
+<!-- jQuery -->
+  <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script> 
+<!-- Bootstrap JS -->
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.bundle.min.js"></script>
+ 
```

Comment:
Asp net core mvc have a defaul bootstrap and jquery in Layout.cshtml

Reply:
I uploaded it because jquery was used in this file, because it didn't work in Layout